app/views/posts/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<!-- this would be to show 1 individual post -->
```

Comment:
So, why is it here?

In addition it could potentially reveal post author for not logged-in user? 🙈
If it would work of course:D